src/workflow/dtos/add-node.input.ts

```diff
@@ -1,12 +1,15 @@
 import { Injectable, PipeTransform } from '@nestjs/common';
-import { ID, Field, InputType, OmitType } from '@nestjs/graphql';
+import { ID, Field, InputType, OmitType, Float } from '@nestjs/graphql';
 import { WorkflowNode, WorkflowNodeState } from '../models/node.model';
 import { DampLabServicePipe } from '../../services/damplab-services.pipe';
 
 @InputType()
 export class AddNodeInput extends OmitType(WorkflowNode, ['_id', 'service', 'state'] as const, InputType) {
   @Field(() => ID, { description: 'The ID of the service this node is a part of' })
   serviceId: string;
+
+  @Field(() => Float, { nullable: true, description: 'Snapshot of service price at submission time' })
```

Comment:
So since this class AddNodeInput extends WorkflowNode (the OmitType thing means, "take all the fields in WorkflowNode but minus _id, service, and state), it already has a `price` field because you already added it to the WorkflowNode class. So you don't need to re-add it here.